src/shims/ApiCompat.proj

```diff
@@ -40,6 +41,16 @@
     </Exec>
 
     <Exec Command="$(ApiCompatCmd) &quot;$(NetStandardRefPath)&quot; @&quot;$(ApiCompatResponseFile)&quot; &gt; $(ApiCompatNSBaselineFile)"
+          Condition="'$(TargetGroup)' != 'netfx'"
```

Comment:
TargetGroup will never be netfx when building verticals because it's not a configuration for this project, so you should be able to remove this condition so that it runs for netcoreapp, uap and uapaot